### ISSUE TITLE
Kill terminal randomly doesnt work

### ISSUE THREAD

**Issue by rushfan010** [microsoft/vscode#128432]
Does this issue occur when all extensions are disabled?: Yes/No

Version: 1.58.0 (user setup)
Commit: 2d23c42a936db1c7b3b06f918cde29561cc47cd6
Date: 2021-07-08T06:54:55.083Z
Electron: 12.0.13
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.18362

Steps to Reproduce:

See gif:

https://user-images.githubusercontent.com/87271347/125199214-8b1fd880-e23b-11eb-83da-4c4811dabc42.mp4

**Comment by rushfan010** [comment]
Vscode is just getting extremely buggy, and users are always being forced into updating to the latest version, and being disrupted by things that worked before no longer working

**Comment by Mlocik97** [comment]
have same problem as well as restarting tasks and few other things,... new version (1.58.0) is totally maximally bugged.

**Comment by rushfan010** [comment]
@Mlocik97 Yeah. I was using a 3 months old version, and today I decided to give it a shot to the newer updates, and now everything is broken and I can't even output things to the terminal anymore. My fault being stupid, I could smell the smoke from a mile away!

I guess I just have an exoteric workflow of printing stuff to the terminal and expecting things to be printed.

**Comment by rushfan010** [comment]
Here another nice gif of things not working which worked before: 

https://user-images.githubusercontent.com/87271347/125202778-82cf9980-e24b-11eb-8b29-f3f7a3f2c9bb.mp4

**Comment by rushfan010** [comment]
> Here another nice gif of things not working which worked before:
> 
>  2021-07-11-13-24-02.mp4 
> Who's responsible for doing terminal stuff, it used to be @Tyriar?

On 1.55.2 it works.

**Comment by Mlocik97** [comment]
I normally have VM, where I have snapshots where I test it, and when it's
OK, I update on my system too.

ne 11. 7. 2021 o 18:16 rushfan010 ***@***.***> napísal(a):
